src/utils/outputs/style-dictionary/colors.ts

```diff
@@ -0,0 +1,149 @@
+import { DefaultColorType, HeisslufType, SpeakingName } from "../../data.ts";
+import { getPalette } from "../index.ts";
+import { setObjectByPath } from "./index.ts";
+
+export const getSDColorPalette = (
+  allColors: Record<string, DefaultColorType>,
+  luminanceSteps: number[],
+): any => {
+  const palette: Record<string, HeisslufType[]> = getPalette(
+    allColors,
+    luminanceSteps,
+  );
+  const colors: any = {};
+
+  Object.entries(allColors).forEach(([unformattedName, color]) => {
+    const name = unformattedName.toLowerCase();
+
+    const colorValues: any = {};
+
+    const hslType: HeisslufType[] = palette[unformattedName];
+    hslType.forEach((hsl) => {
+      colorValues[`${hsl.index ?? hsl.name}`] = { value: hsl.hex };
+    });
+
+    colorValues.origin = {
+      base: {
+        value: color.origin,
+      },
+      light: {
```

Comment:
**nitpick (non-blocking)**: I'd love to see the color mode discriminator as top level property: `colors.dark.origin.base` to make it easier to parse in the style dictionary formatter. 

To derive the actual token name we would need to:

```ts
const styleDictionaryTokenName = "colors.dark.origin.base"
const tokenName = styleDictionaryTokenName.replace(/$colors\.(dark|light)\./, "")
// => "origin.base"
```

Which looks easier than where as if the color mode discriminator is deeper in the token:

```ts
const styleDictionaryTokenName = "colors.origin.dark.base"
const tokenName = styleDictionaryTokenName.replace(/$colors\./, "").split(".").filter(part => ["dark", "light"].includes(part)).join(".") 
// => "origin.base"
```

Reply:
I swapped the structure of the name and the color theme :)